resources/repos.json

```diff
@@ -1096,5 +1096,23 @@
                 "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#23\">RC-23</a>"
             }
         }
+    },
+    "Sefiraat/Simple-Storage:master": {
+        "sonar": {
+            "enabled": true
+        },
```

Comment:
A few problems with this:

First off, you are missing the required settings for sonar integration, see other repositories:
```javascript
sonar: {
    enabled: true,
    organization: "slimefun",
    project-key: "Slimefun_Slimefun4",
    host-url: "https://sonarcloud.io"
},
```

Secondly, Sonarcloud requires access token, so my personal account's access token would require access to your sonarcloud repository which is kinda ehh...
So for now this feature only really works for organizations I am a member of, such as `@Slimefun` or `@Slimefun-Addon-Community` (which you could technically join and move to).
Though for your personal projects, sonar is not an option this way, unless you create a fork of this repository for example.

But I would be happy to merge this without sonar support too if that is okay for you.

Reply:
Disabled sonar for the time being and will revisit should things change with the projects. Thanks :)